### ISSUE TITLE
Missing camera poses, intrinsics, and depthmaps in infer.py

### ISSUE THREAD

**Issue by ncc2k3** [Tencent-Hunyuan/HunyuanWorld-Mirror#28]
Hi, thanks for sharing this great work!

While reading the paper, it is mentioned that the model takes camera poses, intrinsics, and depth maps as inputs for multi-view 3D generation.
However, after checking the infer.py file, it seems that these inputs are not loaded or passed to the model.
Here’s the relevant section:

```
# 3) Load and preprocess images
views = {}
imgs = prepare_images_to_tensor(img_paths, target_size=args.target_size, resize_strategy="crop").to(device)
views["img"] = imgs
B, S, C, H, W = imgs.shape
cond_flags = [0, 0, 0]
print(f"📸 Loaded {S} images with shape {imgs.shape}")

# 4) Inference
with torch.no_grad():
    with torch.amp.autocast('cuda', enabled=bool(use_amp), dtype=amp_dtype):
        predictions = model(views=views, cond_flags=cond_flags)
```

Can you help provide code or clarify this problem.
Thanks for your time and clarification!

**Comment by yifliu3** [comment]
Hi there, thanks for reaching out!

Great question! Currently, our `infer.py` script uses raw images as input by default. However, the model does support incorporating additional priors (such as depth maps, segmentation masks, etc.).

**How to use priors:**
- Please refer to the code snippet in our [README.md](link-to-readme) for a quick example of how to feed priors into the model
- The process is straightforward and requires minimal code changes

**Future updates:**
We're working on adding a complete example script to demonstrate this functionality more clearly. Stay tuned!

**Related discussions:**
For more context and community discussions on this topic, please see #13.

**Comment by qiuzidian** [comment]
@yifliu3 Is it currently unsupported to have partial view conditions? For example, some views have no depth conditions but have intrinsic and extrinsic parameters, while some views have depth conditions but no intrinsic and extrinsic parameters.

**Comment by yifliu3** [comment]
Yes, during training we controlled each type of prior (e.g., depth, intrinsics, extrinsics) to be either present for all views or absent for all views. We did not explicitly train with partial view conditions where some views have certain priors while others do not.
That said, the model may still exhibit some emergent capability for partial-view inference due to its learned representations. Feel free to experiment with partial conditions on your own. It might work to some extent, though we haven't systematically evaluated this scenario.
